src/backend/database/user/addNewCv.ts

```diff
@@ -0,0 +1,28 @@
+import { ref, set } from "firebase/database";
+import { database } from "../../config";
+import { collections } from "../constants";
+import { initializeResumeInstance } from "../resume";
+import { getAuth } from "firebase/auth";
+import { getUserCvs } from "./getUserCvs";
+
+export const addNewCv = async () => {
+  const auth = getAuth();
+  const userId = auth.currentUser?.uid;
+
+  return await getUserCvs().then((cvIds) => {
```

Comment:
If you would've returned null you would not need all those verification below cvIds.val()